### DISCUSSION TITLE
"--save-mot" cannot be used when the specific configuration source is an input video stream.

### DISCUSSION THREAD

**Discussion by runhuayao** [mikel-brostrom/yolo_tracking#1009] [Q&A]
**1.**"--save-mot" cannot be used when I want to use a video stream, such as a *.mp4 file, as the source. There will be an issue of not finding the file save path "exp".
**2.**There is another issue, when I reconstruct the two-dimensional trajectory using the data in the labels file, I need to compare the fitting of the original trajectory in the video with the reconstructed trajectory in order to correct my code. That's why I mentioned above that I want to use the video stream as the input file for "--save-mot". 
**!!!**Thank you very much for implementing the MOT feature, I am really excited about it, even though it can only be applied to a single object at the moment.

**Comment by mikel-brostrom** [comment]
> There will be an issue of not finding the file save path "exp".

Fixed.

Btw, trajectory plotting is a feature I have in the backlog. Should be implemented in the coming weeks :smile:.
